doc/user-guide/cli.md

```diff
@@ -88,6 +88,14 @@ Then, execute:
 ploomber-cloud deploy
 ```
 
+**Optional Configuration**:
+You can use the `--config` flag with the `ploomber-cloud init --from-existing` command to specify a custom name for the configuration file instead of the default `ploomber-cloud.json`. When using this flag, only the file name will be considered, regardless of any specified path. For example:
```

Comment:
Please add a reference to this [section](https://docs.cloud.ploomber.io/en/latest/user-guide/cli.html#switching-the-configuration-file)

Reply:
✅ Added https://github.com/ploomber/doc/pull/287/commits/7d7b32d0e79282703e6f7886cc22c2fe7e19cdcf
✅ Fix Warning block rendering https://github.com/ploomber/doc/pull/287/commits/7cc4de51fd63ac9ec661d323459499896333b27e

Reply:
This is not clear: `..and any specified path is ignored.`. Also, I meant in this section you can add a link something like, `Refer to [this section](https://docs.cloud.ploomber.io/en/latest/user-guide/cli.html#switching-the-configuration-file) to learn more about switching configurations. You can look at other files to see how we link sections from the same file.

Reply:
Just to be sure, is this what you had in mind ? https://github.com/ploomber/doc/pull/287/commits/b525cfdda29b6b7e7c6c4de7505f2baf8da47ecd 
- Precise that --config can but use with a path. not just a name
- Refer to the config section to know more about switching the config file